website/docs/docs/get-started-dbt.md

```diff
@@ -0,0 +1,77 @@
+
+dbt Cloud enables teams with diverse skillsets build reliable data products, at any scale, with capabilities including:
+
+- Development experiences tailored to multiple personas (in-browser [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) or locally with the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation))
+- Out-of-the-box [CI/CD workflows](/docs/deploy/ci-jobs)
+- The [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) for consistent metrics that can be delivered to any end-point
+- Domain ownership of data with multi-project [dbt Mesh](/best-practices/how-we-mesh/mesh-1-intro) setups
+- [dbt Explorer](/docs/collaborate/explore-projects) for collaborative data discovery and understanding
+
+Learn more about [dbt Cloud features](/docs/cloud/about-cloud/dbt-cloud-features) and [start your free trial](https://www.getdbt.com/signup/) today.
+
+<div className="grid--3-col">
+
+<Card
+    title="Quickstart for dbt Cloud and BigQuery"
+    body="Discover how to leverage dbt Cloud with BigQuery to streamline your analytics workflows."
+    link="https://docs.getdbt.com/guides/bigquery"
+    icon="bigquery"/>
+
+<Card
+    title="Quickstart for dbt Cloud and Databricks"
+    body="Learn how to integrate dbt Cloud with Databricks for efficient data processing and analysis."
+    link="https://docs.getdbt.com/guides/databricks"
+    icon="databricks"/>
+
+<Card
+    title="Quickstart for dbt Cloud and Microsoft Fabric"
+    body="Explore the synergy between dbt Cloud and Microsoft Fabric to optimize your data transformations."
+    link="https://docs.getdbt.com/guides/microsoft-fabric"
+    icon="fabric"/>
+
+<Card
+    title="Quickstart for dbt Cloud and Redshift"
+    body="Learn how to connect dbt Cloud to Redshift for more agile data transformations."
+    link="https://docs.getdbt.com/guides/redshift"
+    icon="redshift"/>
+
+<Card
+    title="Quickstart for dbt Cloud and Snowflake"
+    body="Unlock the full potential of using dbt Cloud with Snowflake for your data transformations."
+    link="https://docs.getdbt.com/guides/snowflake"
+    icon="snowflake"/>
+
+<Card
+    title="Quickstart for dbt Cloud and Starburst Galaxy"
+    body="Leverage dbt Cloud with Starburst Galaxy to enhance your data transformation workflows."
+    link="https://docs.getdbt.com/guides/starburst-galaxy"
+    icon="starburst"/>
+
+</div>
+
+### dbt Core
+[dbt Core](/docs/core/about-core-setup) is a command-line [open-source tool](https://github.com/dbt-labs/dbt-core) that enables data teams to transform data using analytics engineering best practices. It's suitable for ‘single players’ and small technical teams who prefer manual setup.
+
+Refer to the following quickstarts to get started with dbt Core:
+
+- [dbt Core from a manual install](/guides/manual-install) to learn how to install dbt Core and set up a project.
```

Comment:
Can we give Core a couple of cards, too? When folks are scanning, based on the Cloud section, that's what they'll look for

Reply:
i think it might look weird to have 3 column grids and then 2 on the same page. going to keep it as is for simplicity purposes